tests/phpunit/includes/Fake_Authentication_Trait.php

```diff
@@ -0,0 +1,71 @@
+<?php
+/**
+ * Trait Google\Site_Kit\Tests\Fake_Authentication_Trait
+ *
+ * @package   Google\Site_Kit\Core\Util
+ * @copyright 2020 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Tests;
+
+/**
+ * Trait for getting fake authentication credentials.
+ *
+ * @since n.e.x.t
+ * @access private
+ * @ignore
+ */
+trait Fake_Authentication_Trait {
```

Comment:
I like it! Maybe in the future we can add some other utility methods here to also take care of user authentication and verification, so that we can fake the entire setup, which may be useful for some tests (e.g. permissions rely on all of that).